eldritch/items.py

```diff
@@ -238,8 +239,36 @@ def DreadCurse(idx):
   return CombatSpell("Dread Curse", idx, {"magical": 9}, 2, -2, 2)
 
 
-class EnchantWeapon(CombatSpell):
+class BindMonster(CombatSpell):
+  def __init__(self, idx):
+    super().__init__("Bind Monster", idx, {}, 2, 4, 2, )
+    self.combat_round = None
+
+  def get_required_successes(self, state):
+    self.combat_round = state.event_stack[-3]
```

Comment:
A minor nitpick here. I would recommend setting combat_round equal to `state.event_stack[-2].combat_round`. In get_usable_interrupt, you correctly validate that this is a combat choice and that the combat choice has a combat round, so I would make self.combat_round point to the object that you know exists and is a combat round (maybe the event stack will look different in the future?).